pkg/reader/ccip.go

```diff
@@ -1425,5 +1425,24 @@ func (r *ccipChainReader) GetMedianDataAvailabilityGasConfig(
 	return daConfig, nil
 }
 
+func (r *ccipChainReader) GetLatestPriceSeqNr(ctx context.Context) (uint64, error) {
+
+	var latestSeqNr uint64
+	err := r.contractReaders[r.destChain].ExtendedGetLatestValue(
```

Comment:
Lets check that this contract reader is available first, like the other methods.